src/Core/Addon/Module/ModuleManager.php

```diff
@@ -71,15 +73,24 @@ class ModuleManager implements AddonManagerInterface
      */
     private $translator;
 
+    /**
+     * @var Employee Legacy employee class
+     */
     private $employee;
+    
+    /**
+     * @var EventDispatcherInterface
+     */
+    private $dispatcher;
 
     public function __construct(AdminModuleDataProvider $adminModulesProvider,
         ModuleDataProvider $modulesProvider,
         ModuleDataUpdater $modulesUpdater,
         ModuleRepository $moduleRepository,
         ModuleZipManager $moduleZipManager,
         TranslatorInterface $translator,
-        Employee $employee = null)
+        Employee $employee = null,
+        EventDispatcherInterface $dispatcher = null)
```

Comment:
In witch cases the dispatcher should be null ? I like a lot your PR but I need to be sure to rely on events in Symfony actions :)